drivers/bridge/bridge.go

```diff
@@ -70,6 +70,7 @@ type networkConfiguration struct {
 	dbIndex            uint64
 	dbExists           bool
 	Internal           bool
+	ifaceManaged       bool
```

Comment:
This is not being persisted. I think that if we reload the daemon, the existing bridge networks will all be marked as non-managed.  So if we delete one, the respective linux bridge will not be removed.